source/usage/tools/brushes.rst

```diff
@@ -128,6 +128,15 @@ Lower Brush
 
 A special case of the deform brush which uses the expression ``y+=1``.
 
+Biome brush
+-------------
+
+::
+
+    /brush biome <shape> [radius] <biomeType>
+
+Sets the biome within the area given by the shape and radius. Keep in mind that since 1.15 the biomes are 3D and the smallest area you can change is a 4x4x4 cuboid. Changing the biome in a large area can be slow. Effects can't be seen until you rejoin the world.
```

Comment:
> Changing the biome in a large area can be slow.

I don't really think this needs to be mentioned, it's much faster than blocks and brushes are usually small anyway.

Reply:
Yeah, you're right. I wrote this after my experiences with changing biomes in large areas with /setbiome, when I expected that it would be a very fast operation.